discovery_test.go

```diff
@@ -261,7 +261,18 @@ func TestGossipSubDiscoveryAfterBootstrap(t *testing.T) {
 
 	// Wait for network to finish forming then join the partitions via discovery
 	for _, ps := range psubs {
-		waitUntilGossipsubMeshCount(ps, topic, partitionSize-1)
```

Comment:
I moved this here becasuse waitUntilGossipsubMeshCount was reported unused by staticcheck (the test is skipped), and this was the only place where it was used. I'd be happy to move it back for clarity  if needed.

Reply:
just comment it out -- also isn't there any way to selectively disable the (imo worthless) tool?